### DISCUSSION TITLE
Directory does not exist or not accessible

### DISCUSSION THREAD

**Discussion by MeowRay** [music-assistant/support#651] [Q&A]
Directory does not exist or not accessible
And the account logged in to nas has read and write permissions
![image](https://user-images.githubusercontent.com/29674595/178499112-f02a6b4a-4208-42e0-8e57-55ffc9c4d896.png)

![image](https://user-images.githubusercontent.com/29674595/178499173-a86c5cc0-eda2-43f1-afaa-d43936a25006.png)
![image](https://user-images.githubusercontent.com/29674595/178499779-2a97343e-630b-4001-ab61-b6ba114c969d.png)
![image](https://user-images.githubusercontent.com/29674595/178499265-d9b1ec80-b4d0-42f1-9dcb-20b22ff469a4.png)

**Comment by OzGav** [comment]
Relates to v1
